src/app/containers/PageHandlers/withData/index.test.jsx

```diff
@@ -49,13 +52,15 @@ describe('withData HOC', () => {
   };
 
   describe('with no data', () => {
+    suppressPropWarnings(['data.dials', 'undefined']);
```

Comment:
This test isnt testing that dials arnt provided, I think you could just add `dials` to the objects above